docs/source/upgrade-guide.rst

```diff
@@ -3,6 +3,23 @@ Upgrade Guide
 
 This upgrade guide lists all breaking changes in plone.restapi and explains the necessary steps that are needed to upgrade to the lastest version.
 
+
+Upgrading to plone.restapi X.y
+------------------------------
+
+All versions before plone.restapi x.y.z are affected by an issue that
+converts ids of reordered content to unicode from a bytestring when running on
+python2.
+
+- Only content that was reordered is affected.
```

Comment:
Thinking of it, this could also do with some clarification. We should make it easy and unambigous for people to determine whether or not they need to run the fix or not. Maybe something like this:

```
You may be affected by this issue and should run the fix if:

- You used the PATCH "ordering" functionality of plone.restapi
- Were using Python 2 at that point
- Are seeing issues with objectIds() returning mixed string types
```